testgrid/config.yaml

```diff
@@ -23,6 +23,7 @@ default_test_group:
   alert_stale_results_hours: 0 # Don't alert for staleness by default.
   num_failures_to_alert: 3 # Consider a test failed if it has 3 or more consecutive failures.
   num_passes_to_disable_alert: 1 # Consider a failing test passing if it has 1 or more consecutive passes.
+  code_search_path: github.com/kubernetes/kubernetes/search # URL for regression search links.
```

Comment:
two questions:
- is this the previous value?
- `https://`? or no

Reply:
depending on how exactly this is used, there are also other search endpoints, but I suppose that can be in  a follow-up

Reply:
Yup, this is the current hardcoded default internally; I'm just moving it to be in the external config instead. (Shouldn't need 'https://' since it works already.)

If a group/tab needs to specify a different path they can overwrite it in their config, so that should be okay. If this is about having multiple potential defaults, that should be handled separately yeah. ^^

Reply:
sweet